pkg/fanal/analyzer/analyzer.go

```diff
@@ -142,8 +142,9 @@ type AnalysisInput struct {
 }
 
 type PostAnalysisInput struct {
-	FS      fs.FS
-	Options AnalysisOptions
+	FS           fs.FS
+	Options      AnalysisOptions
+	FilePatterns []string
```

Comment:
Is this filed for file patterns? It looks like matched file paths.

Reply:
Updated filed name in https://github.com/aquasecurity/trivy/pull/7365/commits/5f493acb3671d7a2f53980b6e8a84c0236832613